src/Cake.Common/Tools/MSTest/MSTestRunner.cs

```diff
@@ -60,6 +60,19 @@ private ProcessArgumentBuilder GetArguments(FilePath assemblyPath, MSTestSetting
                 builder.AppendQuoted("/noisolation");
             }
 
+            // Got a specific test settings file?
+            if (!string.IsNullOrWhiteSpace(settings.TestSettingsFile))
+            {
+                // Add the file as a property.
+                var testSettingsFile = settings.TestSettingsFile.Trim();
+                if (testSettingsFile.Contains(" "))
+                {
+                    testSettingsFile = testSettingsFile.Quote();
```

Comment:
Always quote the settings file regardless of whether or not it contains whitespace (this is how we've done it elsewhere).